sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 5.10.0-beta.1 (Unreleased)
+## 5.9.2 (2023-06-06)
```

Comment:
Just noticed that we are not bumping the minor version - don't we normally do this for any API changes?